src/Microsoft.AspNet.Mvc.Core/Rendering/Html/TemplateBuilder.cs

```diff
@@ -5,9 +5,9 @@
 using Microsoft.AspNet.Mvc.ModelBinding;
 using Microsoft.Framework.Internal;
 
-namespace Microsoft.AspNet.Mvc.Rendering
+namespace Microsoft.AspNet.Mvc.Rendering.Internal
```

Comment:
looks like this project contains a few `internal` namespaces already and they follow the pattern `Microsoft.AspNet.Mvc.Internal.XYZ`.  so these should instead be in `Microsoft.AspNet.Mvc.Internal.Rendering`.

files should also be moved to the `src/Microsoft.AspNet.Mvc.Core/Internal/Rendering` folder.